### ISSUE TITLE
Webhook

### ISSUE THREAD

**Issue by rubendinho** [JCBarry/argyle#1]
Thanks for putting this together.  I might be missing something here, but is there a good way to provide a webhook? 

Thanks,
Ruben

**Comment by JCBarry** [comment]
Hey Ruben!
Happy to help.  You weren't missing something, the gem didn't support this.

I just released a new version (0.1.0) which supports the `token` and `webhook` optional params.  You can use it like this:

``` ruby
<%= plaid_link(:name => 'Tony Soprano', :action => my_callback_path, :token => 'myToken', :webhook => 'http://meathook.com') %>      
```

**Comment by rubendinho** [comment]
Nice! Thanks Jason.
